File: falleaves-server/falleaves-starter-model/src/main/java/org/techas/falleaves/model/UserInfoEntity.java
package org.techas.falleaves.model;

import javax.persistence.*;
import java.sql.Timestamp;

@Entity
@Table(name = "user_info", schema = "falleaves", catalog = "")
public class UserInfoEntity {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    private Long uid;
    private String nickname;
    private String email;
    private String phone;
    private Long age;
    private String location;
    private java.sql.Timestamp birthday;


    public Long getId() {
        return id;
    }

    public UserInfoEntity setId(Long id) {
        this.id = id;
        return this;
    }

    public Long getUid() {
        return uid;
    }

    public UserInfoEntity setUid(Long uid) {
        this.uid = uid;
        return this;
    }

    public String getNickname() {
        return nickname;
    }

    public UserInfoEntity setNickname(String nickname) {
        this.nickname = nickname;
        return this;
    }

    public String getEmail() {
        return email;
    }

    public UserInfoEntity setEmail(String email) {
        this.email = email;
        return this;
    }

    public String getPhone() {
        return phone;
    }

    public UserInfoEntity setPhone(String phone) {
        this.phone = phone;
        return this;
    }

    public Long getAge() {
        return age;
    }

    public UserInfoEntity setAge(Long age) {
        this.age = age;
        return this;
    }

    public String getLocation() {
        return location;
    }

    public UserInfoEntity setLocation(String location) {
        this.location = location;
        return this;
    }

    public Timestamp getBirthday() {
        return birthday;
    }

    public UserInfoEntity setBirthday(Timestamp birthday) {
        this.birthday = birthday;
        return this;
    }
}

File: falleaves-server/auth/src/main/java/cn/falleaves/auth/model/UserInfoEntity.kt
package cn.falleaves.auth.model

import javax.persistence.*

@Entity
@Table(name = "user_info", schema = "falleaves", catalog = "")
open class UserInfoEntity {

    @get:Id
    @get:Column(name = "id", nullable = false)
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    open var id: Int? = null

    @get:Basic
    @get:Column(name = "nickname", nullable = true)
    open var nickname: String? = null

    @get:Basic
    @get:Column(name = "email", nullable = true)
    open var email: String? = null

    @get:Basic
    @get:Column(name = "phone", nullable = true)
    open var phone: String? = null

    @get:Basic
    @get:Column(name = "age", nullable = true)
    open var age: Byte? = null

    @get:Basic
    @get:Column(name = "location", nullable = true)
    open var location: String? = null

    @get:Basic
    @get:Column(name = "birthday", nullable = true)
    open var birthday: java.sql.Timestamp? = null

    @get:Basic
    @get:Column(name = "createAt", nullable = true)
    open var createAt: java.sql.Timestamp? = null

    @get:Basic
    @get:Column(name = "updateAt", nullable = true)
    open var updateAt: java.sql.Timestamp? = null
}

